problems/[230608] Number of Islands.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * leetcode problem link: https://leetcode.com/problems/number-of-islands
+ *
+ * @param {character[][]} grid
+ * @return {number}
+ */
+const numIslands = function (grid) {
+  let islandCount = 0;
+
+  for (let row = 0; row < grid.length; row++) {
+    for (let column = 0; column < grid[row].length; column++) {
+      if (grid[row][column] === "1") {
+        islandCount += findCliff(row, column);
+      }
+    }
+  }
```

Comment:
```suggestion
  for (let row = 0; row < grid.length; row++) {
    for (let column = 0; column < grid[row].length; column++) {
      islandCount += findCliff(row, column);
    }
  }
```

해당 좌표가 "0"인지 판별하는 로직은 findCliff에도 있으니 중복 실행될 것 같아요~ 없어도 잘 동작하지 않을까 합니다.